src/Service.Tests/Authorization/AuthorizationResolverUnitTests.cs

```diff
@@ -1282,6 +1423,56 @@ private static RuntimeConfig BuildTestRuntimeConfig(EntityPermission[] permissio
             );
             return runtimeConfig;
         }
+
+        /// <summary>
+        /// Returns a ClaimsPrincipal object created from a jwt token which uses either the provided
+        /// claimsCollection or a default claimsCollection capturing various value types of possible claims.
+        /// Value types include:
+        /// - space delimited string for scp claim
+        /// - JSON String array (roles, groups)
+        /// - JSON object (distributed groups claim)
+        /// - JSON int/bool array (theoretical, though not in EntraID access token)
+        /// </summary>
+        /// <param name="userClaims">Test configured collection of claims to include in ClientPrincipal</param>
+        /// <returns>Authenticated ClaimsPrincipal with user specified or default claims</returns>
+        private async static Task<ClaimsPrincipal> CreateClaimsPrincipal(Dictionary<string, object?>? userClaims = null)
+        {
+            if (userClaims is null)
+            {
+                Dictionary<string, object> distributedClaimSources = new()
+                {
+                    ["src1"] = new { endpoint = "https://graph.microsoft.com/v1.0/users/{userID}/getMemberObjects" }
+                };
+
+                userClaims = new()
+                {
+                    { "scope", "scope1 scope2 scope3" },
+                    { "groups", "src1" },
```

Comment:
`groups` isnt an array here as the function comment expects.

Reply:
updated comment to no longer state 'groups' is an array. it is covered by being described as a "distributed groups claim" and "json object"